lifecycleScripts/install.js

```diff
@@ -93,19 +95,24 @@ function build() {
 
     opts.env.HOME = path.join(home, ".atom-shell-gyp");
 
+    runtime = "--runtime=electron";
+
     target = "--target=" + asVersion;
 
     distUrl = "--dist-url=https://gh-contractor-zcbenz.s3." +
       "amazonaws.com/atom-shell/dist";
   }
   else if (nwVersion) {
-    builder = "nw-gyp";
+    runtime = "--runtime=node-webkit";
```

Comment:
You should be able to nuke the builder argument entirely and just hardcode it to node-pre-gyp I think.